src/cr/cube/cubepart.py

```diff
@@ -707,23 +707,18 @@ def population_counts(self):
         """2D np.float64 ndarray of population counts per cell.
 
         The (estimated) population count is computed based on the `population` value
-        provided when the Slice is created. It is also adjusted to account for any
-        filters that were applied as part of the query.
-
-        If any of the dimensions (rows or columns) is a categorical date, the
-        appropriate percentages are used, to calculate the population counts, so as to
-        be the total amount among categorical dates.
+        provided when the Slice is created (`._population`). It is also adjusted to
+        account for any filters that were applied as part of the query
+        (`._cube.population_fraction`).
 
-        Otherwise, table percents are used to calculate population counts.
+        `._population` and `_cube.population_fraction` are both scalars and so do not
+        affect sort order.
         """
-        proportions = (
-            self.row_proportions
-            if self.rows_dimension_type in (DT.CAT_DATE, DT.CA_SUBVAR)
-            else self.column_proportions
-            if self.columns_dimension_type in (DT.CAT_DATE, DT.CA_SUBVAR)
-            else self.table_proportions
+        return (
+            self._assembler.cell_population_fractions
```

Comment:
The naming choice `.cell_population_fractions` is a little unfortunate here. In what way is it related to `_cube.population_fraction` which shares the same root name? I think the answer is "it's really a different concept", but I'll read on.

----

Okay, you'll see below, I recommend `.population_proportions` for this name. It's still the familiar proportions concept, one of row, column, or table, just that which one is appropriate for computing population depends on a few things.